data/json/npcs/robofac/NPC_ROBOFAC_INTERCOM.json

```diff
@@ -189,7 +188,29 @@
       "type": "dialogue",
       "context": "intercom",
       "value": "yes",
-      "yes": "So, do you need something?",
+      "yes": [
+        "So, do you need something?",
+        "Yes, mercenary?",
+        "Mercenary.",
+        "Mercenary.",
```

Comment:
Double string here is intended?

Reply:
Yeah Its just to make that particular response more frequent.